napari_tiff/_tests/test_tiff_reader.py

```diff
@@ -1,14 +1,13 @@
+import numpy as np
 import os
+import pytest
+import tifffile
 import zipfile
 
-import numpy as np
 from napari_tiff import napari_get_reader
 from napari_tiff.napari_tiff_reader import (imagecodecs_reader,
-                                            imagej_reader,
                                             tifffile_reader,
                                             zip_reader)
-import pytest
-import tifffile
```

Comment:
Why did you remove the imagej_reader from tests?

Reply:
Oh, I see, you're trying to just unify everything under one reader that automagically does the right thing. That's fine, but I think it's worth keeping and testing the other readers as independent functions. This makes it easier to understand when one particular functionality is broken.

Reply:
This PR removes imagej_reader entirely, and instead has separate functions for finding imagej/ome/tiff metadata (instead of having separate functions for reading each of those types of tiff files).

Reply:
Ok it also saves imagej metadata - I didn't know that was supported. I'll put back the test.